### PR TITLE
fix(Webhook): unknown error in catch block

### PR DESCRIPTION
**Pull request by jpbberry**

### CODE REVIEW

File: src/structs/Webhook.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ export class Webhook {
           res.sendStatus(204);
         }
       } catch (err) {
-        this.options.error?.(err);
```
Comment (Xetera) [inline src/structs/Webhook.ts:129]: I'm confused, what exactly is getting thrown from functions that isn't an error? If we're handling all errors with a 500 response, shouldn't they also be treated as an error on the library side? Maybe the correct move here should be to change the type of the error callback from `Error` to `unknown`

**Review by Xetera — COMMENTED** [review]
Sorry for the long wait for the review :disappointed:

File: src/structs/Webhook.ts (hunk 1 above)
Comment (jpbberry) [reply src/structs/Webhook.ts:129]: Are we sure we want to do this? It would be a breaking change. I'm not opposed but a major bump seems unnecessary.

Comment (Xetera) [reply src/structs/Webhook.ts:129]: Well, this is also a breaking change technically, just more subtle

**Review by Xetera — APPROVED** [review]
Alright let's just get this in and remember to do the type change for next major release

**Pull request merged**